tests/test_snap_point.py

```diff
@@ -20,7 +20,7 @@ def test_data():
 
 
 @hypothesis.given(
-    nudge=st.tuples(st.floats(-20, 20), st.floats(-5, 5), st.floats(-20, 20))
+    nudge=st.tuples(st.floats(-10, 10), st.floats(-2.5, 2.5), st.floats(-10, 10))
```

Comment:
Hypothesis was complaining about the search rejecting too many points, so I made it slightly narrower (this is still much larger than the mesh so it'll search everywhere).